src/components/AddItems.js

```diff
@@ -1,9 +1,101 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import db from '../firebase/firebase';
+import './AddItems.css';
+
+const firebase = require('firebase/app');
```

Comment:
It doesn't seem like you need lines 5 and 6 here.
Since you are importing the db from ../firebase/firebase.
There is no need for these imports. I tried removing them locally and the app still worked. 
Is there a reason why you added this?

Reply:
You are correct! We had imported it for an earlier iteration where we called methods that we aren't using anymore. I have another question. I see that there is a `firebase.js `file in the lib folder (where it's exported as` fb`) and a` firebase.js` in the firebase folder (where it's exported as `db `- the one shayne & I created last week).

Reply:
I wasn't aware of the file in lib.
From what I see the best place for the db would be in lib/firebase.js instead of in firebase/firebase.js
I would suggest moving the firebase/firebase.js to the lib folder and getting rid of the current one in lib.
This will avoid confusion going forward.